R/thematic.R

```diff
@@ -461,8 +462,8 @@ sequential_gradient <- function(fg_weight = 0.75, bg_weight = 0.5, fg_low = TRUE
   if (n < 3) {
     stop("`n` must be 3 or more.", call. = FALSE)
   }
-  if (!is.logical(fg_low)) {
-    stop("`fg_low` must be `TRUE` or `FALSE`", call. = FALSE)
+  if (!is.logical(bg_high) && !identical(bg_high, "auto")) {
+    stop("`bg_high` must be either `'auto'`, `TRUE` or `FALSE`", call. = FALSE)
```

Comment:
```suggestion
    stop("`bg_high` must be either `'auto'`, `TRUE`, or `FALSE`", call. = FALSE)
```
I subscribe to the oxford comma